src/app/services/user/user.service.ts

```diff
@@ -19,41 +18,19 @@ export interface User {
   providedIn: 'root'
 })
 export class UserService {
-  private users: Observable<User[]>;
-  private userCollection: AngularFirestoreCollection<User>;
   private staticUserCollection: AngularFirestoreCollection<User>;
-
-  subscription: any;
+  private userCollection: AngularFirestoreCollection<User>;
+  private staticUsers: Observable<User[]>;
+  private users: Observable<User[]>;
 
   filteredUsers: Observable<User[]>;
 
   constructor(private afs: AngularFirestore) {
-    console.log('Auth Service constructor');
+    console.log('User Service Start');
+    this.staticUserCollection = this.afs.collection<User>('static_users');
     this.userCollection = this.afs.collection<User>('users');
-    this.users = this.userCollection.snapshotChanges().pipe(
```

Comment:
These changes must be a result of the extensive _observables_ learning you were doing, eh?

Reply:
This is technically from learning the differences between the different getters provided by AngularFire. Read the difference between `.subscribe(...)`, `.valueChanges(...)`, and `.snapshotChanges(...)`.

https://github.com/angular/angularfire/blob/master/docs/firestore/documents.md
https://github.com/angular/angularfire/blob/master/docs/firestore/collections.md